canonical-web-and-design/jaas-dashboard
# Add Usabilia link to primary nav

## Done

Add Usabilia to primary nav

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click the link in the sidebar and verify pop-up

## Details

Fixes: https://github.com/canonical-web-and-design/juju-squad/issues/1076
